expose addImportMap (#2429)
systemjs/systemjs: docs/import-maps.md

Previous versions of the import maps spec had support for multiple import maps in a single web page ([explanation](https://github.com/WICG/import-maps/issues/199)). SystemJS added support for multiple import maps during that time and has decided to keep support for multiple import maps as an experimental feature ([discussion](https://github.com/systemjs/systemjs/issues/2095)). Note that the Chrome implementation of import maps does not yet allow for multiple maps, and use of multiple import maps within SystemJS should be considered experimental and subject to change.

### Dynamically add Import Maps

> Standard import map is based on dom operation, for SystemJS, you can use `System.addImportMap` to add import map dynamically for browser and node.

Reviewer: Perhaps something like:

```suggestion
To dynamically add import maps, a non-spec API is available to extend additional mappings into the import map at any time:
```

Previous versions of the import maps spec had support for multiple import maps in a single web page ([explanation](https://github.com/WICG/import-maps/issues/199)). SystemJS added support for multiple import maps during that time and has decided to keep support for multiple import maps as an experimental feature ([discussion](https://github.com/systemjs/systemjs/issues/2095)). Note that the Chrome implementation of import maps does not yet allow for multiple maps, and use of multiple import maps within SystemJS should be considered experimental and subject to change.

### Dynamically add Import Maps

> > **Note** Non-standard Extension
> > 
> > This may override existing import maps, for more details, please check https://github.com/guybedford/es-module-shims#overriding-import-map-entries

Reviewer: ```suggestion
> Non-standard Extension
```

> To dynamically add import maps, a non-spec API is available to extend additional mappings into the import map at any time:

Reviewer: ```suggestion
The `addImportMap` method is available to dynamically extend additional mappings into the import map at any time:
```

```js
System.addImportMap({
  "imports": {
    "y": "/path/to/y.js",
  }
})

> ```

Reviewer: ```suggestion
```

Any existing map entries will be overridden with the new values.
```

> 

Reviewer: ```suggestion

```suggestion

Any existing map entries will be overridden with the new values.
```
